test/fixtures/xruby/exercises/alpha/example.tt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
```

Comment:
I thought there might have been when I created it, but it turned out further letters weren't needed.